spec/cell_spec.rb

```diff
@@ -36,20 +36,19 @@ def cell(options={})
     end
 
     it "should convert nil, Numeric, and Date values to strings" do
-      [nil, 123, 123.45, Date.today].each do |value|
+      [nil, 123, 123.45, Date.today, Time.new].each do |value|
         c = @pdf.cell(:content => value)
         expect(c).to be_a_kind_of Prawn::Table::Cell::Text
         expect(c.content).to eq value.to_s
       end
     end
 
-    it "should allow inline styling with a hash argument" do
-      # used for table([[{:text => "...", :font_style => :bold, ...}, ...]])
-      c = Prawn::Table::Cell.make(@pdf,
-                                  {:content => 'hello', :font_style => :bold})
-      expect(c).to be_a_kind_of Prawn::Table::Cell::Text
-      expect(c.content).to eq "hello"
-      expect(c.font.name).to eq 'Helvetica-Bold'
```

Comment:
Why did you remove this spec?

Reply:
That was unintentional.  I'll put it back.